app/javascript/topics/components/TopicsShow__PostShow.re

```diff
@@ -210,7 +212,11 @@ let make =
       ~archivePostCB,
     ) => {
   let user = findUser(users);
```

Comment:
@harigopal : Regarding the last comment on the `Post.edited` function, this was done so because `editorId` could be null for having a deleted user. So such a concrete check is needed.

Reply:
@mahesh-krishnakumar Ooh, okay. So that the text can saying something like _"Edited by Deleted User"_.